eyre/tests/test_location.rs

```diff
@@ -140,7 +142,7 @@ fn test_option_compat_wrap_err() {
     }));
 
     use eyre::ContextCompat;
-    let err = None::<()>.wrap_err("oopsie").unwrap_err();
+    let err = None::<()>.context("oopsie").unwrap_err();
```

Comment:
The encosing test `test_option_compat_wrap_err` might better be renamed to `test_option_compat_context`?

Reply:
`test_option_compat_context` and `test_option_compat_with_context` already exist and are (now) identical. The tests `test_option_compat_wrap_err` and `test_option_compat_wrap_err_with` are obsolete and should be deleted.

Reply:
And this is why consistent naming is so good, cause now that yields a duplicate definition error.

Thanks a lot for your suggestions 😊